tests/utils/nodes/nodes.go

```diff
@@ -144,3 +149,81 @@ func LabelNode(client corev1Typed.CoreV1Interface, node *corev1.Node, label, val
 
 	return err
 }
+
+// EnableMasterScheduling enables/disables master nodes scheduling.
+func EnableMasterScheduling(client corev1Typed.CoreV1Interface, scheduleable bool) error {
```

Comment:
This function is no longer needed and was removed by https://github.com/test-network-function/cnfcert-tests-verification/pull/437

Reply:
Still needed for Kind cluster setups.  The calls to these funcs are wrapped in the `IsKindCluster` func.